lib/content-services/src/lib/content-node-selector/content-node-selector.component.ts

```diff
@@ -179,16 +198,18 @@ export class ContentNodeSelectorComponent implements OnInit {
     }
 
     getWarningMessage(): string {
-        return this.showingSearch ? 'NODE_SELECTOR.UPLOAD_BUTTON_SEARCH_WARNING_MESSAGE' :
-            (this.hasNoPermissionToUpload() ? 'NODE_SELECTOR.UPLOAD_BUTTON_PERMISSION_WARNING_MESSAGE' : '');
+        if (this.showingSearch) {
+            return 'NODE_SELECTOR.UPLOAD_BUTTON_SEARCH_WARNING_MESSAGE';
+        } else {
+            return this.hasNoPermissionToUpload() ? 'NODE_SELECTOR.UPLOAD_BUTTON_PERMISSION_WARNING_MESSAGE' : '';
```

Comment:
Small thing but this can be outside else because inside if you have return: 
```ts
if (this.showingSearch) {
  return 'NODE_SELECTOR.UPLOAD_BUTTON_SEARCH_WARNING_MESSAGE';
}
return this.hasNoPermissionToUpload() ? 'NODE_SELECTOR.UPLOAD_BUTTON_PERMISSION_WARNING_MESSAGE' : '';
```